Source/DafnyLanguageServer/Workspace/VerificationProgressReporter.cs

```diff
@@ -379,14 +394,38 @@ IDiagnosticPublisher diagnosticPublisher
   }
 
   /// <summary>
-  /// Returns the verification priority for a given token.
+  /// Returns the verification priority for a given method, depending on if it was recently (last 5 edits)
   /// </summary>
-  /// <param name="token">The token to consider</param>
+  /// <param name="method">The method to consider</param>
   /// <returns>The automatically set priority for the underlying method, or 0</returns>
-  private int GetVerificationPriority(IToken token) {
+  private int GetVerificationPriority(VerificationTree? method) {
+    if (method != null) {
+      var lastTouchedIndex = document.LastTouchedMethodPositions.IndexOf(method.Position);
+      // 0 if not found
+      if (lastTouchedIndex == -1) {
+        return 0;
+      }
+      var lastTouchedCount = document.LastTouchedMethodPositions.Count;
+      // 10 if last edited.
+      var priority = 11 + lastTouchedIndex - lastTouchedCount;
```

Comment:
Could you put the `11` into a const variable and explain the motivation for its value?